loopchain/blockchain/exception.py

```diff
@@ -125,6 +125,12 @@ class AnnounceNewBlockError(Exception):
     message_code = message_code.Response.fail_announce_block
 
 
+class RoundMismatchError(Exception):
+    """
+    """
+    pass
+
+
```

Comment:
I strongly agree with you to define exceptions. But I think that the name has no similarity between other exceptions (`BlockVersionNotMatch`, BlockHeightMismatch`, ...).

How about consider setting a standard Exception names, and fit the class to it?